.github/workflows/documentation.yml

```
@@ -0,0 +1,35 @@
+name: deploy documentation
+
+on:
+  push:
+    branches: [master]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    env:
+      AWS_ACCESS_KEY_ID: "vAMX1aNVpxXrzSYKc2zjotsizyLPSrjvEqCsjRfadDh2CaIV+bF4EoRMFXDvyl/unb9GXxlJ5KTCgZ2cW+8HLsHxda4SYl0f03Du8/F4XbSa9JOyiwdzSW3Du/cpDxITIrZGMY/wKx3q6diGeKnX8+FjN3BAnIfdYCBtPeyisL/jjzjOxMh6xpDWADixej7sBZeRwpmL/PkQUickFabGCtfbxzdpH8l/dEd116D0eL/xivZqvAWTTB77nC/lWs1w6x+x5M7Q0Ch+AcSAzy8N4jFVJSMcxJMhqil4EoNjXFwBHNoMKKoJ8X9833js8l2gVfPRZ4mme26z9zwsSgRvDarrjH2/k5yzgMZV95QLsOqo7HADB2s3sRvAPEpl+13UO74LVpt5vfZ+6Z8rqhIZFGyVF8Qj3Glv9Bt2mJl8XF728lgdZMwNZZLg3dXqJQ9qzkAoub5VMAI8HtiaUaa3dHVW+nHKuCabzZtgcNgkRa/cr4F/h+gAUDzo1otDxWrjxTgoK5Grf8Nop5YPOe7oKuSIpOlOKh25ZVbFmldrTdfXzTqYbICGhSJYiu59rozViLszNCN/cRmfmrhhEcVZh+i55eODmVnThk/7jC6JPsUcBWcLq4Zc4BoUMiAUPhiHXtoTylgdERlcKQ7KznIvpkS1u8U3z1P8TSbu8eB43tU="
+      AWS_SECRET_ACCESS_KEY: "e4GO9KglnmjVwufPq5i0hzpCt40If7IKD6m3Qer3QeBal6x5blFoDQdXL7lqJ/WIdmeSF22p580SdTdlYRtEYulYaQqQs3MohggqtxRF2J268aSB32tTpwaa8WP1qpPv6Trzb93Tky2SwZOO1c82lI80/iONYaCRkc80J8gX2ebzHfhklCnN1xAQ0k68dFgJPitywtEwwatpgHnfvYFVe3B/rjdpUg1CItheWmazxIzwHc5evB2+vk0+KGWE2a7hosfiMdlgiKfcbFAV0J6UgP+qY4eAK0ekCEL3xB8DlpcjQaLYPQUK/9k0pkdUl+ma78a9XrikO+RR5Ul6unm42bC1ZV4BpCUFGLNw/T27dqT/Ot9laOKdWohR9DSmJxJ0Cd0MbxeeLlPobdu35ev5u0MuhKz8pJQoGPMXkT0KK2YIP6q7/dDd0ZIAHd6LN1nhMpXyR1ZqYoJxqhE0c5/V0HZ+I1NSeM+FQuig+qq8D0DwP8WNU0FprFFHjNBW7rOkxjYw1fMqHwWbtWXkj+SvxE3ktzycf9Ja66fMFcFferi3XtW01YEh7e2sFG5BhdwM3AMH5e0m3V0KhoCCckBXYn6thmg8PcKzICs2k3fRvhL2eJygb+d0AIZqarUY+gAegDAbP+6x/eOzcXGs+7q9tGMt9Pof+BVgKTmszJz9TIM="
+
+    steps:
+      - name: Setup PHP
+        uses: shivammathur/setup-php@v2
+        with:
+          php-version: '8.1'
+
+      - name: Checkout Code
+        uses: actions/checkout@v2
+
+      - name: Build phar with makefile
+        run: ./scripts/run_doctum_docs_on_github.sh
+
+      - name: Deploy
+        uses: reggionick/s3-deploy@v3
+        with:
+          folder: doctum-output/build/
+          bucket: rokka-io
+          bucket-region: eu-central-1
+          invalidation: /
+          delete-removed: false
```

https://github.com/marketplace/actions/s3-deploy says true here, but not sure if that bucket is shared with other things.

---

```suggestion
          delete-removed: true
```
we seem to either use a key that points to the right folder, or the doc resides in its own bucket.